rector.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+
+return RectorConfig::configure()
+    ->withPaths([
+        __DIR__ . '/config',
+        __DIR__ . '/routes',
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+    ])
+    // uncomment to reach your current PHP version
+    //->withPhpSets(php80: true)
```

Comment:
Am I right in assuming that we'd set the current PHP version as 8.1 here since that's the minimum version required for the package? 🙂

Reply:
Yes, I tried it, but it raised some other issues we need to address. So, to keep this specific PR small, I started with two simple rulesets. One of the great things of Rector is that you can introduce it in a project with just a few rules and then add more rules later, with a kind of "progressive" approach